applications/ParticleMechanicsApplication/custom_utilities/mpm_search_element_utility.h

```diff
@@ -81,7 +81,7 @@ namespace MPMSearchElementUtility
             }
             for (size_t i = 1; i < cross_products.size(); ++i)
             {
-                if (cross_products[i] * cross_products[0] < 0.0)
+                if (cross_products[i] * cross_products[0] < -std::abs(Tolerance))
```

Comment:
This is actually more restrictive - I think you want a positive tolerance
```suggestion
                if (cross_products[i] * cross_products[0] < std::abs(Tolerance))
```

Reply:
I think it should be negative tolerance, because in the if-scope the variable `is_sinide` is set to false, am I right?

Reply:
Sorry, gotchya, you're right!